base/utils/database.go

```diff
@@ -4,27 +4,17 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"log"
-	"os"
 
 	"github.com/jackc/pgx/v4"
 	_ "github.com/jackc/pgx/v4/stdlib" // Needed to make pgx work with database/sql
 )
 
-func getDbVar(envVar string) string {
-	value, ok := os.LookupEnv(envVar)
-	if !ok {
-		log.Fatalf("Unable to get env var: %s.\n", envVar)
-	}
-	return value
-}
-
 func GetDbURL() string {
-	dbHost := getDbVar("POSTGRES_HOST")
-	dbPort := getDbVar("POSTGRES_PORT")
-	dbName := getDbVar("POSTGRES_DB")
-	dbUser := getDbVar("POSTGRES_USER")
-	dbPassword := getDbVar("POSTGRES_PASSWORD")
+	dbHost := GetEnv("POSTGRES_HOST", "localhost")
```

Comment:
If we are introducing default values here, it should be `vuln4shift_database` and `5432` right?

Reply:
oh, I have tested it locally and forgot to change, will fix